docs/README.md

```diff
@@ -0,0 +1,65 @@
+# KILI SDK docs
+
+The Kili SDK doc is created with [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), a theme for MkDocs which is a static site generator geared towards technical project documentation.
+The webiste is hosted in Github Pages [here](https://kili-technology.github.io/kili-playground/)
+
+## install dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+## HTML Preview
+
+Automatically reloads when you edit a file.
+
+```
+mike serve
+```
+
+## List all versions
+
+```
+mike list
+```
+
+## create a new version
+
+```
+mike deploy [version: X.XX] [alias]...
```

Comment:
which Github rights does it require? (this should be available to a restricted set of internal users only)

Reply:
This will only make changes locally.
if a user wants to push the changes, it can add the --push argument. This will push the changes to a `gh-pages` branch and it is from here that the Github Pages will build the website. I guess that the repo does not allow external people to push to a specific branch ?

Reply:
no they can't, so we should be good.

Reply:
yes I do validate.